### PR TITLE
Import UX improvements

### PR DESCRIPTION
**Pull request by Bios-Marcel**
* Technical
  * import now in seperate location
  * import now returns a summary instead of just possible duplicates
* UI/UX
  * summary is now displayed as a dialog after importing
  * file chooser
    * now allows to pick multiple files
    * now displays which account files are being imported for

### CODE REVIEW

**Comment by bugabinga** [comment]
Import summary has 4 categories now:
1. Successful
2. Duplicates
3. Potential Duplicates
4. Skipped
It is not immediatly clear if:
- those are exclusive groups
- potential duplicates get imported or not

**Comment by Bios-Marcel** [comment]
>     * those are exclusive groups
Yes!
>     * potential duplicates get imported or not
No! "Potential Duplicates" isn't new, but was the only existing group before. Everything else was siltently discarded / imported. The behaviour SHOULD not have changed.

File: src/main/java/link/biosmarcel/baka/view/PaymentsView.java [hunk 1]
```diff
@@ -182,44 +184,63 @@ private void classifyAllUnclassified() {
...
         final FileChooser fileChooser = new FileChooser();
+        fileChooser.setTitle("Import for account '%s'".formatted(account.name));
+
         final var initialDir = state.data.convenienceState.importDirectories.get(account);
         if (initialDir != null) {
             fileChooser.setInitialDirectory(new File(initialDir));
         } else if (state.data.convenienceState.lastImportPath != null) {
             fileChooser.setInitialDirectory(new File(state.data.convenienceState.lastImportPath));
         }
-        final var file = fileChooser.showOpenDialog(getTabPane().getScene().getWindow());
```
Comment (bugabinga) [inline src/main/java/link/biosmarcel/baka/view/PaymentsView.java:191]: getScene will be `null`, when the Node ,TabPane in this case, is not yet part of the scene graph. that can happen in tests or during constructors usually. is this a risk in this case?

**Review by bugabinga — APPROVED** [review]

File: src/main/java/link/biosmarcel/baka/view/PaymentsView.java (hunk 1 above)
Comment (Bios-Marcel) [reply src/main/java/link/biosmarcel/baka/view/PaymentsView.java:191]: No risk here

**Pull request merged**